app/controllers/posts_controller.rb

```diff
@@ -18,53 +18,48 @@ def add_profile_username
     end
     if @profile == nil
       respond_to do |format|
-        format.html { redirect_to council_posts_path(@council), flash: {alert: 'Hittade ingen användare med det användarnamnet.'}}
+        format.html { redirect_to council_posts_path(@council), flash: {alert: 'Hittade ingen användare med det användarnamnet.'} }
       end
-    elsif @profile.name.blank?          
-      redirect_to council_posts_path(@council), flash: {alert: 'Användaren :"' + @user.username.to_s + '" måste fylla i fler uppgifter i sin profil.'} 
+    elsif @profile.name.blank?
+      redirect_to council_posts_path(@council), flash: {alert: 'Användaren :"' + @user.username.to_s + '" måste fylla i fler uppgifter i sin profil.'}
```

Comment:
Line is too long. [150/100]<br>Space inside { missing.<br>Space inside } missing.